tools/aqrdev

```diff
@@ -348,17 +348,18 @@ def cmd_start(ctx: AppCtx, name: str, conservative: bool) -> None:
 
 
 @app.command("stop")
+@click.option("-f", "--force", flag_value=True)
 @click.argument("name", required=True, type=str)
 @pass_appctx
-def cmd_stop(ctx: AppCtx, name: str) -> None:
+def cmd_stop(ctx: AppCtx, name: str, force: bool) -> None:
     """
     Stop deployment; destroys running machines
     """
     deployment: Deployment = _get_deployment(ctx, name)
     assert deployment
 
     try:
-        deployment.stop()
+        deployment.stop(not force)
```

Comment:
What do you think about using this?
```suggestion
        deployment.stop(interactive=not force)
```

Reply:
for better readability I suggest to use:
```
if force:
  deployment.stop(interactive=false)
else:
  deployment.stop()

Reply:
Though, there is difference, non-interactive mode does abort if some question happens during execution, when force, it forces the answer with default option, so interactive is not the same as 'not force'.

Reply:
@votdev used the suggested parameter assignment with the new `force` parameter.